src/module/account/settings/SettingFormLayout.tsx

```diff
@@ -0,0 +1,117 @@
+'use client';
+
+import { FormButtonToolbar, FormContainer } from '@/shared/components/form/FormElements';
+import { Controller, useForm } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { Button } from '@/shared/components/Button';
+import * as z from 'zod';
+import FormInput from './_component/FormInput/FormInput';
+import FormPasswordInput from './_component/FormInput/FormPasswordInput';
+
+const formSchema = z.object({
+  displayName: z
+    .string()
+    .min(1, { message: 'Display name is required' })
```

Comment:
be careful the magic string here, since you're gonna take the translation ticket, I won't ask for a change here, these string will be further extracted to somewhere else in that task

Reply:
added error messages enum in the shared/utils/helpers